cmd/experimental/experimental.go

```diff
@@ -14,22 +14,20 @@
 //
```

Comment:
GitHub is drunk. `cmd/experimental/experimental.go` is a new file, not a move from `cmd/io.go`. That one has been deleted.